average.js

```diff
@@ -1,6 +1,11 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  // filters all the numbers which are not Nan and stores them in nanListNumbers
+  const nanListNumbers = numbers.filter((number) => {
```

Comment:
We name variables, so that readers can understand their intent without needing to read the entire code.
Here, `nanListNumbers` doesn't give the understanding that they are numbers _excluding_ NaN.
Try clarifying the variable name.